scaletest/terraform/action/gcp_db.tf

```diff
@@ -0,0 +1,89 @@
+resource "google_sql_database_instance" "db" {
+  name                = "${var.name}-coder"
+  project             = var.project_id
+  region              = local.deployments.primary.region
+  database_version    = "POSTGRES_14"
```

Comment:
This was the default value when it was previously a tf variable, I removed the variable in favor of a hard code here since this changes infrequently and not between different scenarios.